kafka11/src/main/java/zipkin2/reporter/kafka11/KafkaSender.java

```diff
@@ -230,6 +230,7 @@ AdminClient getAdminClient() {
     if (closeCalled) return;
     KafkaProducer<byte[], byte[]>  producer = this.producer;
     if (producer != null) producer.close();
+    if (adminClient != null) adminClient.close();
```

Comment:
since this is a volatile ref, the safe way is to make a local variable like the producer. then you don't have a risk of a NPE race!